src/generated-types.ts

```diff
@@ -2532,22 +3344,76 @@ export interface paths {
     };
   };
   "/utils/txs/evaluate": {
-    /** Submit an already serialized transaction to evaluate how much execution units it requires. */
+    /**
+     * Submit a transaction for execution units evaluation 
+     * @description Submit an already serialized transaction to evaluate how much execution units it requires.
+     */
     post: {
+      /**
+       * Submit a transaction for execution units evaluation 
+       * @description Submit an already serialized transaction to evaluate how much execution units it requires.
+       */
       parameters: {
         header: {
           "Content-Type": "application/cbor";
         };
       };
       responses: {
         /**
-         * Returns result of EvaluateTx function from Ogmios see
+         * @description Returns result of EvaluateTx function from Ogmios see
+         * <a href="https://ogmios.dev/mini-protocols/local-tx-submission/#evaluatetx">EvaluateTx</a>
+         * and <a href="https://ogmios.dev/api/">API reference (EvaluateTxResponse)</a> for related errors.
+         */
+        200: {
+          content: {
+            "application/json": Record<string, never>;
```

Comment:
It used to be `"application/json": { [key: string]: unknown };`.
`Record<string, never>` will most likely cause issues.

We need this.
https://github.com/drwpow/openapi-typescript/pull/1032

Reply:
workaround 1dc8320a7e1d57ade9cbb0819d9214723ec9ebaf